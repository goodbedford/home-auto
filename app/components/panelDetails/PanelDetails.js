import React from "react";


const PanelDetails = (props) => {

  return (
    <div className="home-auto__panel-controls">
        {props.children}
    </div>
  );
};

export default PanelDetails;
