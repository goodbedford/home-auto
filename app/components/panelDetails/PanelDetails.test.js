/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import PanelDetails from "./PanelDetails";

describe("<PanelDetails />", () => {

  it("PanelDetails snapshot", () => {
    const wrapper = shallow(<PanelDetails />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
