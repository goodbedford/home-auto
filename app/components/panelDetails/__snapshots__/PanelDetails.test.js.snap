// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PanelDetails /> PanelDetails snapshot 1`] = `
<div
  className="home-auto__panel-controls"
/>
`;
